Makefile

```diff
@@ -503,14 +503,14 @@ gen-doc =	\
 		else \
 			cd tools/doc && node ../../$(NPM) install; \
 		fi;\
-	[ -x $(NODE) ] && $(NODE) $(gen-json) || node
+	[ -x $(NODE) ] && $(NODE) $(1) || node
```

Comment:
I think if you simply remove `$(gen-json)`/`$(1)` you won't need the change in the other too lines
(this will simply resolve to the path of the `node` exe)